primeqa/mrc/README.md

```diff
@@ -236,6 +236,23 @@ xlm-roberta-large -> NQ Lists: Minimal F1 = 46.95
 xlm-roberta-large -> PrimeQA/tydiqa-primary-task-xlm-roberta-large -> NQ Lists: Minimal F1 = 57.44
 ```
 
+### Task Arguments
+
+Some task arguments take references which allow for dynamic imports of existing or
+user-defined functionality.  For example, to select the `ExtractivePostProcessor` use
+`--postprocessor primeqa.mrc.processors.postprocessors.extractive.ExtractivePostProcessor`.
+Alternatively, a new postprocessor could be written and selected with 
+`--postprocessor qualified.path.to.new.postprocessor.NewPostProcessor`.
+
+For example, if one was implementing a new model which made predictions by means other than
+an extractive head then a `NewPostProcessor` which derived predictions from the model
+outputs would be needed.
+
+Similarly, when adding support for a new dataset (with a new schema) a new preprocessor would be needed.
+This would be selected by specifying `--preprocessor qualified.path.to.new.postprocessor.NewPreProcessor`
+for the `NewPreProcessor` corresponding to this dataset and schema.
+
+
 ### PrimeQA also supports answering questions over tables
```

Comment:
Moving the task arguments above the table qa section does not look right to me. TableQA is a special form of MRC and should be together with the boolean and list qa. Task arguments section should be the last one, and perhaps it needs more work. Might need to be expanded or the name should be changed, but this is outside this PR.

Reply:
@mihaela-bornea : We re-organized the content based on @seirasto's suggestion.